config/environments/production.rb

```diff
@@ -55,7 +55,7 @@
   # config.cache_store = :mem_cache_store
 
   # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  # config.action_controller.asset_host = "http://assets.example.com"
+  config.action_controller.asset_host = "http://comfy.moneyadviceservice.org.uk"
```

Comment:
I've swapped the quotes and add the missing test environment config. I'm not sure how using .env.dev will work. There is no corresponding .env file on QA nor are the all the environment variables found that file present in configuration management (scripts repo)